File: 백준/Bronze/2480. 주사위 세개/주사위 세개.cc
#include <iostream>

int main(){
    
    int a,b,c = 0;
    std::cin>>a>>b>>c;
     

    if (a==b && b==c){
        std::cout<<10000+a*1000;
    } else if (a==b && b!=c){
        std::cout<<1000+a*100;
    } else if (a==c && b!=c){
        std::cout<<1000+a*100;
    } else if (c==b && b!=a){
        std::cout<<1000+c*100;
    } else if (a!=b && b!=c){
        // if (a>b && a>c){
        //     std::cout<<a*100;
        // } else if (b>a && b>c){
        //     std::cout<<b*100;
        // } else if (c>a && c>b){
        //     std::cout<<c*100;
        // }
        int m = 0;

        m = std::max(a,std::max(b,c));
            std::cout<<m*100;
    }
    
    
    return 0;
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
a, b,c = map(int,input().split())

if a==b and b==c:
    print(10000+a*1000)
elif a==b and b!=c:
    print(1000+a*100)
elif a==c and b!=c:
    print(1000+a*100)
elif c==b and b!=a:
    print(1000+c*100)
elif a!=b and b!=c:
    m = 0
    m = max(a,b,c)
    print(m*100)
